src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/cosmic/CosmicFuncotationFactory.java

```diff
@@ -300,6 +303,18 @@ public String getName() {
         return outputFuncotations;
     }
 
+    private void updateProteinChangeCountMap(final Map<String, Integer> proteinChangeCounts, final ResultSet resultSet) {
+        final String proteinChange = getProteinChangeStringFromResults(resultSet);
+        if ( !proteinChange.isEmpty() ) {
+            if ( proteinChangeCounts.containsKey(proteinChange) ) {
+                proteinChangeCounts.put(proteinChange, proteinChangeCounts.get(proteinChange) + 1);
```

Comment:
There is a better method to use to do this:  

```
final int count = proteinChangeCounts.getOrDefault(proteinChange, 0);
proteinChangeCounts.put(proteinChange, count+1);
```

Reply:
Fixed!